torch/distributed/rpc/api.py

```diff
@@ -33,7 +33,8 @@
     PythonUDF,
     RPCExecMode,
     _internal_rpc_pickler,
-    _start_record_function,
+    _profiling_enabled,
+    build_rpc_profiling_key,
```

Comment:
any reason this does not start with an underscore? Is this intended to be a user-facing function?

Reply:
Will change it to be underscore